packages/paste-core/components/side-panel/stories/components/SidePanelWithContent.tsx

```diff
@@ -32,38 +31,25 @@ export const SidePanelWithContent: React.FC<React.PropsWithChildren> = () => {
       <Separator orientation="horizontal" verticalSpacing="space0" />
       <SidePanelBody>
         <Box width="100%">
-          <ChatLog>
-            <ChatMessage variant="inbound">
-              <ChatBubble>Hello, what can I help you with?</ChatBubble>
-              <ChatMessageMeta aria-label="said by Gibby Radki at 3:35 PM">
-                <ChatMessageMetaItem>
-                  <Avatar name="Gibby Radki" size="sizeIcon20" />
-                  Gibby Radki ・ 3:35 PM
-                </ChatMessageMetaItem>
-              </ChatMessageMeta>
-            </ChatMessage>
-            <ChatMessage variant="outbound">
-              <ChatBubble>Hi! What is your return policy?</ChatBubble>
-              <ChatMessageMeta aria-label="said by you at 3:35 PM">
-                <ChatMessageMetaItem>3:35 PM</ChatMessageMetaItem>
-              </ChatMessageMeta>
-            </ChatMessage>
-            <ChatMessage variant="inbound">
-              <ChatBubble>Hello, what can I help you with?</ChatBubble>
-              <ChatMessageMeta aria-label="said by Gibby Radki at 3:35 PM">
-                <ChatMessageMetaItem>
-                  <Avatar name="Gibby Radki" size="sizeIcon20" />
-                  Gibby Radki ・ 3:35 PM
-                </ChatMessageMetaItem>
-              </ChatMessageMeta>
-            </ChatMessage>
-            <ChatMessage variant="outbound">
-              <ChatBubble>Hi! What is your return policy?</ChatBubble>
-              <ChatMessageMeta aria-label="said by you at 3:35 PM">
-                <ChatMessageMetaItem>3:35 PM</ChatMessageMetaItem>
-              </ChatMessageMeta>
-            </ChatMessage>
-          </ChatLog>
+          <AIChatLog>
```

Comment:
todo: Adjust either Side Panel or AI Chat Log so there's not padding being added by both things. See this comment: https://github.com/twilio-labs/paste/pull/3827#discussion_r1608948217

Reply:
removed side padding from AI Chat Log. decided not to remove from Side Panel because the scrollbar takes up some space on the right side of the container and it would get a little funky with adjusting for that

Reply:
let's jam on this later today. We can resolve all the scrollbar + spacing issues.